pykit/Command.py

```diff
@@ -0,0 +1,20 @@
+class Command:
+    """
+    Object for calling an IBM i CL command.
+    """
+    def __init__(self, command, screen_output=False):
+        self.command = command
+        self.screen_output = screen_output
+        self.payload = {
+            "cmd": {}
+        }
+
+    def get_payload(self):
+        if self.screen_output:
+            self.payload["cmd"]["qsh"] = self.command
+        elif any(x in self.command for x in ["&", "?"]):
```

Comment:
Good enough for now, but note that this will cause unexpected results:

```python
Command("CALL QUESTION 'What is the meaning of life?'")
Command("ENDSBS SBS(MYSBS) OPTION(*IMMED) /* End the subsystem & do it immediately */")
```
May want to put a comment in to that effect.

Reply:
Can  you elaborate a little more, please? I'm not sure I understand.

Reply:
You are checking whether either '&' or '?' are inside the string, but those characters could be inside an innocuous string or comment inside the CL command string. `MYCL ARG('?')` is different than `MYCL ARG(?)`. In the first, we're passing a single character to the `ARG` parameter on input, while the second is getting an output parameter.